releasenotes/notes/incremental-log-manager-0e7bfc306f58570f.yaml

```diff
@@ -0,0 +1,11 @@
+features:
+  - |
+    Enable log manager for log caching
+
+fixes:
+  - |
+    Upload cached logs in time intervals
+    The log manager uploads the cached contents of taskout.log and
+    harness.log every 15 seconds. This allows to follow task progress
+    and avoids missing logs when the external watchdog aborts the
+    recipe.
```

Comment:
This write-up is for the user.  It wouldn't hurt to mention how the user can change or disable the timing if needed.

Reply:
We already discussed that we don't want to disable the log manager, the feature is for debugging purposes only, as mentioned in the PR message. If the log manager needs to be disabled, that will be because some corner case that we missed, and it should be fixed. The feature can be used for testing and workaround without downgrading.
And the same goes for the upload interval, the default value should be low enough to follow the progress of the task. Given the constraints we have doesn't make sense to change it.

Reply:
I can imagine how it will end up if we will tell them that they can disable logmanager.
Every single automation that is creating job XML for them will include it.. :)